### DISCUSSION TITLE
No Next.js version could be detected in your project.

### DISCUSSION THREAD

**Discussion by 12emir** [vercel/next.js#26612] [Help]
Hi there! i have a problem with deploying to vercel. I also tried with newest version of next

The error is:
Error! No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"

<img width="666" alt="Screenshot 2021-06-25 at 18 27 00" src="https://user-images.githubusercontent.com/25539419/123456431-1dd93a00-d5e3-11eb-9d5b-7e0119ed12a2.png">

<img width="402" alt="Screenshot 2021-06-25 at 18 27 14" src="https://user-images.githubusercontent.com/25539419/123456468-29c4fc00-d5e3-11eb-8470-24d78e8fea48.png">

**Comment by Messa57fr** [comment]
Hi, I have the same problem :( Did you find out? 
Error: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"

Thanks
